src/vs/code/electron-main/windows.ts

```diff
@@ -709,7 +709,18 @@ export class WindowsManager implements IWindowsService {
 		}
 
 		// Handle folders to open
-		let openInNewWindow = openConfig.preferNewWindow || openConfig.forceNewWindow;
+		let openInNewWindow: boolean;
```

Comment:
@elemongw sorry, but this does not work for me when I try it. The folder opens in the same window no matter how I configure the setting. I believe you need to turn the logic around: set `openInNewWindow` to true when the setting is enabled!